backend/app/tests/utils/utils.py

```
@@ -18,7 +18,7 @@
 
 @pytest.fixture(scope="class")
 def openai_credentials():
-    settings.OPENAI_API_KEY = "sk-fake123"
+    OPENAI_API_KEY = "sk-fake123"
```

Ruff (0.12.2)

Local variable `OPENAI_API_KEY` is assigned to but never used

Remove assignment to unused variable `OPENAI_API_KEY`

(F841)